Turkey/TestOutputFormat.cs

```diff
@@ -31,7 +36,7 @@ public async override Task AfterRunningTestAsync(string name, TestResult result,
                     switch (result)
                     {
                         case TestResult.Passed: resultOutput = "PASS"; break;
-                        case TestResult.Failed: resultOutput = "FAIL"; break;
+                        case TestResult.Failed: resultOutput = "FAIL"; failedTests.Add($"{string.Format("{0,-60}", name)}[{resultOutput}]\t({elapsedTime})"); break;
```

Comment:
Have you considered not adding the extra spaces? What about leaving out "FAIL", since this is a list of tests that failed?

Reply:
I have no issue with the FAIL being printed again and preserving the format as before.
I mostly care about the elapsed time being there.